### DISCUSSION TITLE
[Nuxt 3.12.4] nuxt.config.ts file nitro.hooks["render: html"] is invalid

### DISCUSSION THREAD

**Discussion by 275923543** [nuxt/nuxt#28703] [General]
// nuxt.config.ts
export default defineNuxtConfig({
  nitro: {
    hooks: {
      "render:html"() {
         // do something
      }
    }
  }
})

"render:html"() is invalid, why?

**Comment by cuebit** [comment]
Do you mean `"render:html": () => {}`?
